src/Features/DiagnosticsTestUtilities/CodeActions/CSharpCodeRefactoringVerifier`1+Test.cs

```diff
@@ -51,6 +51,7 @@ static Test()
             public Test()
             {
                 _sharedState = new SharedVerifierState(this, DefaultFileExt);
+                this.FixedState.InheritanceMode = StateInheritanceMode.AutoInherit;
```

Comment:
I am highly concerned about this significantly changing the test behavior in ways that cannot be explained.

Reply:
What is the significant change in test behavior.  

> that cannot be explained.

Who can explain it?  Isn't this the behavior of normal CodeFix Tests?  They inherit the end state from the start state?  Which is why you don't need to supply the 'FixedCode' if it is the same as the 'TestCode'.